File: implementations/c/sieve/sieve_run.c
#include "sieve.h"

int main() {
    int upperBounds[5] = {1, 10, 100, 1000, 10000};
    int index = 0;
    for (index = 0; index < 5; index ++) {
        run(upperBounds[index]);
    }

    return 0;
}

File: implementations/java/sieve/sieve_run.java
import java.util.HashMap;

public class sieve_run {

    private static HashMap<Integer, Integer> known = new HashMap<Integer, Integer>();

    public static void main(String[] args) {
        known.put(1, 0);
        known.put(10, 4);
        known.put(100, 25);
        known.put(1000, 168);
        known.put(10000, 1229);

        sieve sieve = new sieve();
        for (int n : known.keySet()) {
            sieve.run(n);
        }
    }
}
